src/app/core/header/header.component.test.js

```diff
@@ -0,0 +1,36 @@
+import { HeaderComponent } from './header.component';
+
+const MAIN_ROUTE = {
+    path: 'dashboard',
+    headerName: 'Dashboard',
+    iconImage: './assets/dashboard.svg',
+    main: true
+};
+const NAVIGATION_ROUTES = [
+    MAIN_ROUTE,
+    { path: 'devices', headerName: 'Devices', iconImage: './assets/deviceList.svg' },
+    { path: 'scenarios', headerName: 'Scenarios', iconImage: './assets/scenarioList.png' },
+    { path: 'help', headerName: 'Help', iconImage: './assets/help.svg' }
+];
+
+describe('Header', () => {
+    let sut;
+
+    beforeEach(() => {
+        sut = new HeaderComponent();
+        sut.routes = NAVIGATION_ROUTES;
+        sut.ngOnInit();
+    });
+
+    it('should be defined', () => {
```

Comment:
I suppose we can remove this test, because in next tests we check this implicitly